libs/sdk-model/api/sdk-model.api.md

```diff
@@ -218,6 +218,9 @@ export const catalogItemMetadataObject: (catalogItem: CatalogItem) => MetadataOb
 // @public
 export type CatalogItemType = "attribute" | "measure" | "fact" | "dateDataset";
 
+// @alpha
+export type ChangeType = "added" | "modified";
```

Comment:
Naming is pretty vague, maybe add some prefix?

Reply:
You mean names for ChangeType or added and modified? :)

Reply:
I agree that `ChangeType` is too vague

Reply:
And on a similar note: `"addedByPlugin"` and `"modifiedByPlugin"`?